File: VNNLanguage/Constants/RegexConstants.cs
using System.Text.RegularExpressions;

namespace VNNLanguage.Constants
{
    public static class RegexConstants
    {
        public static Regex CharacterNameRegex = new Regex(@"\[(.*?)\]");
        public static Regex GetStuffInQuotes = new Regex(@"\""(.*?)\""");
        public static Regex GetPixelValue = new Regex(@"[0-9]*\.?[0-9]+(px|%)?");
        public static Regex GetRoundBracketNumbers = new Regex(@"([0-9\.]+)");
        public static Regex GetStuffInAstrix = new Regex(@"\*(.*?)\*");
        public static Regex GetSprite = new Regex(@"\](.*?)\*");
    }
}

File: SM-engine/src/services/RegexConstants.ts
export const RegexConstants = {
  CharacterNameRegex: /\[(.*?)\]/,
  GetStuffInQuotes: /"(.*?)"/,
  GetPixelValue: /[0-9]*\.?[0-9]+(px|%)?/,
  GetRoundBracketNumbers: /([0-9\.]+)/g,
  GetStuffInAstrix: /\*(.*?)\*/,
  GetSprite: /\](.*?)\*/,
  GetPosition: /\((left|right|centre|center)\)/i,
};
